examples/http/src/main/java/io/opentelemetry/example/http/HttpClient.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.example.http;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.context.propagation.HttpTextFormat;
+import io.opentelemetry.exporters.logging.LoggingExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.TracerSdkFactory;
+import io.opentelemetry.sdk.trace.export.SimpleSpansProcessor;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Status;
+import io.opentelemetry.trace.Tracer;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.charset.Charset;
+
+public class HttpClient {
+
+  // OTel API
+  private static Tracer tracer =
+      OpenTelemetry.getTracerFactory().get("io.opentelemetry.example.http.HttpClient");
+  // Export traces to log
+  private static LoggingExporter loggingExporter = new LoggingExporter();
+  // Inject the span context into the request
+  private static HttpTextFormat.Setter<HttpURLConnection> setter =
+      new HttpTextFormat.Setter<HttpURLConnection>() {
+        @Override
+        public void put(HttpURLConnection carrier, String key, String value) {
+          carrier.setRequestProperty(key, value);
+        }
+      };
+
+  private void initTracer() {
+    // Get the tracer
+    TracerSdkFactory tracerFactory = OpenTelemetrySdk.getTracerFactory();
+    // Show that multiple exporters can be used
+
+    // Set to export the traces also to a log file
+    tracerFactory.addSpanProcessor(SimpleSpansProcessor.newBuilder(loggingExporter).build());
+  }
+
+  private HttpClient() throws Exception {
+    initTracer();
+
+    // Connect to the server locally
+    int port = 8080;
+    URL url = new URL("http://127.0.0.1:" + port);
+    HttpURLConnection con = (HttpURLConnection) url.openConnection();
+
+    // Name convention for the Span is not yet defined.
+    // See: https://github.com/open-telemetry/opentelemetry-specification/issues/270
+    Span span = tracer.spanBuilder("/").setSpanKind(Span.Kind.CLIENT).startSpan();
+    // TODO provide semantic convention attributes to Span.Builder
```

Comment:
File an issue to not forget about this?

Reply:
This comment can be removed since there is not yet a sampling relevant attribute AFAIK. Or am I wrong?